crypto/rsa_extra/rsassa_pss_asn1_test.cc

```diff
@@ -0,0 +1,367 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You have compiled a nice set of tests. I have some suggestions for additional unhappy path tests:
* Switch around sequences: i.e. the object with sequence tag [1] comes before sequence with tag [0] (both not absent)
* Sequence with an invalid explicit tag.
* Sequences contains double values with the same sequence tag
* Sequences have an encoded length, but are actually too long
* Sequences have an encoded length, but are actually too short
* Cases with objects having an encoded length, but are either too long or too short
* Trailer field is not 1

Reply:
> Switch around sequences: i.e. the object with sequence tag [1] comes before sequence with tag [0] (both not absent)
> Sequence with an invalid explicit tag.
> Sequences contains double values with the same sequence tag

Added.

> Cases with objects having an encoded length, but are either too long or too short
> Sequences have an encoded length, but are actually too long
> Sequences have an encoded length, but are actually too short

https://github.com/awslabs/aws-lc/pull/90 validated these two cases.

> Trailer field is not 1

Discussed in another thread. Also, https://github.com/awslabs/aws-lc/pull/90 validated this case.